Issue with typescript 2.2.1

#### Category
[ ] Enhancement

[x ] Bug

[ ] Question

#### Expected / Desired Behavior / Question
All my automated build started failing today as typescript 2.2.1 has been released. I fixed it taking of the ^ in front of the typescript version and fixing it to 2.2.0.

#### Observed Behavior
All builds report the same error message:
```
ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5080324Z (11168,11): error TS2300: Duplicate identifier 'Headers'.
2017-02-22T19:14:12.5080324Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5080324Z (11177,13): error TS2300: Duplicate identifier 'Headers'.
2017-02-22T19:14:12.5080324Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5080324Z (13544,11): error TS2300: Duplicate identifier 'ReadableStream'.
2017-02-22T19:14:12.5080324Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5080324Z (13550,13): error TS2300: Duplicate identifier 'ReadableStream'.
2017-02-22T19:14:12.5080324Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5080324Z (13566,11): error TS2300: Duplicate identifier 'Request'.
2017-02-22T19:14:12.5080324Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5236550Z (13583,13): error TS2300: Duplicate identifier 'Request'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5236550Z (13588,11): error TS2300: Duplicate identifier 'Response'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5236550Z (13599,13): error TS2300: Duplicate identifier 'Response'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5236550Z (19116,18): error TS2451: Cannot redeclare block-scoped variable 'fetch'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5236550Z (19121,6): error TS2300: Duplicate identifier 'BodyInit'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5236550Z (19142,6): error TS2300: Duplicate identifier 'HeadersInit'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\typescript\lib\lib.d.ts
2017-02-22T19:14:12.5236550Z (19152,6): error TS2300: Duplicate identifier 'RequestInfo'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-streams\index.d.ts
2017-02-22T19:14:12.5236550Z (32,15): error TS2300: Duplicate identifier 'ReadableStream'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (11,13): error TS2451: Cannot redeclare block-scoped variable 'fetch'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (13,14): error TS2300: Duplicate identifier 'HeadersInit'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (14,15): error TS2300: Duplicate identifier 'Headers'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (31,14): error TS2300: Duplicate identifier 'BodyInit'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (43,14): error TS2300: Duplicate identifier 'RequestInfo'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (44,15): error TS2300: Duplicate identifier 'Request'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (64,11): error TS2300: Duplicate identifier 'Request'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (88,15): error TS2300: Duplicate identifier 'Response'.
2017-02-22T19:14:12.5236550Z ERROR in C:\agent\_work\12\s\node_modules\@types\whatwg-fetch\index.d.ts
2017-02-22T19:14:12.5236550Z (107,11): error TS2300: Duplicate identifier 'Response'.
```

#### Steps to Reproduce
Set up a project relying on sp-pnp-js and typescript. upgrade to latest versions of all (typescript@2.2.1, sp-pnp-js@2.0.1) and you'll get this error.
I'm building using webpack 2.

--- a/package.json
+++ b/package.json
@@ -5,8 +5,7 @@
   "main": "./lib/pnp.js",
   "typings": "./lib/pnp",
   "dependencies": {
-    "@types/sharepoint": "^2013.0.31",
-    "@types/whatwg-fetch": "^0.0.33"
+    "@types/sharepoint": "^2013.0.31"
   },
   "devDependencies": {
     "@types/chai": "^3.4.34",
@@ -35,7 +34,7 @@
     "node-fetch": "^1.4.1",
     "ts-loader": "^1.2.2",
     "tslint": "^4.4.2",
-    "typescript": "^2.1.0",
+    "typescript": "^2.2.1",
     "webpack": "^1.13.3",
     "webpack-dev-server": "^1.16.2",
     "yargs": "^6.0.0"

--- a/src/net/httpclient.ts
+++ b/src/net/httpclient.ts
@@ -6,10 +6,10 @@ import { APIUrlException } from "../utils/exceptions";
 export interface FetchOptions {
     method?: string;
     headers?: string[][] | { [key: string]: string };
-    body?: BodyInit;
-    mode?: string | RequestMode;
-    credentials?: string | RequestCredentials;
-    cache?: string | RequestCache;
+    body?: any;
+    mode?: "navigate" | "same-origin" | "no-cors" | "cors";
+    credentials?: "omit" | "same-origin" | "include";
+    cache?: "default" | "no-store" | "reload" | "no-cache" | "force-cache" | "only-if-cached";
 }
 
 export class HttpClient {

--- a/src/net/sprequestexecutorclient.ts
+++ b/src/net/sprequestexecutorclient.ts
@@ -1,4 +1,4 @@
-import { HttpClientImpl, FetchOptions } from "./httpclient";
+import { HttpClientImpl } from "./httpclient";
 import { Util } from "../utils/util";
 import { SPRequestExecutorUndefinedException } from "../utils/exceptions";
 
@@ -9,7 +9,7 @@ export class SPRequestExecutorClient implements HttpClientImpl {
     /**
      * Fetches a URL using the SP.RequestExecutor library.
      */
-    public fetch(url: string, options: FetchOptions): Promise<Response> {
+    public fetch(url: string, options: any): Promise<Response> {
         if (typeof SP === "undefined" || typeof SP.RequestExecutor === "undefined") {
             throw new SPRequestExecutorUndefinedException();
         }

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -7,8 +7,7 @@
         "sourceMap": true,
         "removeComments": false,
         "types": [
-            "sharepoint",
-            "whatwg-fetch"
+            "sharepoint"
         ],
         "noImplicitAny": true,
         "allowUnreachableCode": false,

